ref/test/test_output/yppm_0.0.12.test

```diff
@@ -1,20 +0,0 @@
-TEST ===================================================================================================================
```

Comment:
This file should not be empty.  How are the tests passing?

Reply:
Fixed.  Good catch!